Doc/library/sqlite3.rst

```diff
@@ -742,29 +804,40 @@ Connection Objects
 
    .. method:: backup(target, *, pages=-1, progress=None, name="main", sleep=0.250)
 
-      This method makes a backup of an SQLite database even while it's being accessed
-      by other clients, or concurrently by the same connection.  The copy will be
-      written into the mandatory argument *target*, that must be another
-      :class:`Connection` instance.
-
-      By default, or when *pages* is either ``0`` or a negative integer, the entire
-      database is copied in a single step; otherwise the method performs a loop
-      copying up to *pages* pages at a time.
-
-      If *progress* is specified, it must either be ``None`` or a callable object that
-      will be executed at each iteration with three integer arguments, respectively
-      the *status* of the last iteration, the *remaining* number of pages still to be
-      copied and the *total* number of pages.
-
-      The *name* argument specifies the database name that will be copied: it must be
-      a string containing either ``"main"``, the default, to indicate the main
-      database, ``"temp"`` to indicate the temporary database or the name specified
-      after the ``AS`` keyword in an ``ATTACH DATABASE`` statement for an attached
-      database.
-
-      The *sleep* argument specifies the number of seconds to sleep by between
-      successive attempts to backup remaining pages, can be specified either as an
-      integer or a floating point value.
+      Create a backup of an SQLite database.
+
+      :param target:
+          The database connection to save the backup to.
+      :type target: sqlite3.Connection
+
+      :param pages:
+          If equal to or less than ``0``,
+          the entire database is copied in a single step;
+          otherwise the backup is performed *pages* pages at a time.
```

Comment:
```suggestion
          The number of pages to copy at a time.
          If equal to or less than ``0``,
          the entire database is copied in a single step.
```

Clearer and cleaner to frame this in terms of the primary behavior of the parameter, no?